packages/common/rejector.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * @import {AssertionUtilities} from 'ses';
+ */
+
+/**
+ * @typedef {false | AssertionUtilities['Fail']} Rejector;
```

Comment:
I don't understand lint error
```
packages/common/rejector.js:6:3 - [error] Failed to find JSDoc tag for Rejector after parsing comment, please file a bug report
```
Please help. Thanks.

Reply:
The `false |` lets me make the `reject` parameter non-optional.

Separately, I saw no need to weaken the template return type from `=> never` to `=> false`. By not weakening it, the type system may (or may not) help us remember to stick to the pattern 
```js
reject && reject`...`
```
rather that just
```js
reject`...`
```

Reply:
Problem went away.